db/schema.rb

```diff
@@ -27,17 +25,16 @@
   end
 
   create_table "trips", force: :cascade do |t|
```

Comment:
yeah it only shows the stations part as green. and that one I have to change to add date time. what lines are you referring to?